aiogram_dialog/widgets/kbd/calendar_kbd.py

```diff
@@ -159,11 +168,11 @@ def days_kbd(self, offset) -> List[List[InlineKeyboardButton]]:
             weekheader,
             *days,
             [
-                InlineKeyboardButton(text="Prev month",
+                InlineKeyboardButton(text=emoji.emojize(':left_arrow:'),
                                      callback_data=f"{self.widget_id}:{MONTH_PREV}"),
-                InlineKeyboardButton(text="Zoom out",
+                InlineKeyboardButton(text=emoji.emojize(':magnifying_glass_tilted_left:'),
                                      callback_data=f"{self.widget_id}:{SCOPE_MONTHS}"),
-                InlineKeyboardButton(text="Next month",
+                InlineKeyboardButton(text=emoji.emojize(':right_arrow:'),
```

Comment:
You DO NOT NEED `emoji` library to render emoji. Use unicode.
Anyway, probably we need text customization here instead of changing soruces